src/workerd/api/node/zlib-util.c++

```diff
@@ -655,7 +655,7 @@ BrotliEncoderContext::BrotliEncoderContext(ZlibMode _mode): BrotliContext(_mode)
   state = kj::disposeWith<BrotliEncoderDestroyInstance>(instance);
 }
 
-void BrotliEncoderContext::close() {
+BrotliEncoderContext::~BrotliEncoderContext() {
   auto instance = BrotliEncoderCreateInstance(alloc_brotli, free_brotli, alloc_opaque_brotli);
```

Comment:
I meant to ask about this before but forgot... why is the destructor creating a new instance here and assigning to state?

Reply:
I've removed them all. I had an eye opening moment thanks to you. I've removed the destructors.